PS.g4

```diff
@@ -186,7 +186,7 @@ fragment INFTY: '\\infty';
 SYMBOL: PI | INFTY;
 
 fragment VARIABLE_CMD: '\\variable';
-fragment VARIABLE_SYMBOL: (GREEK_LETTER | LETTER | DIGIT)+ (UNDERSCORE (GREEK_LETTER | LETTER | DIGIT)+)?;
+fragment VARIABLE_SYMBOL: (GREEK_LETTER | LETTER | DIGIT)+ (UNDERSCORE ((L_BRACE (GREEK_LETTER | LETTER | DIGIT | COMMA)+ R_BRACE) | (GREEK_LETTER | LETTER | DIGIT)))?;
```

Comment:
So for multiple letters in the subscript it needs to be wrapped in braces? Is that done automatically? Also is there supposed to be a comma in there?

Reply:
Yes, yes, and yes 😄